core/src/dsnp/schema.rs

```diff
@@ -0,0 +1,181 @@
+use crate::dsnp::dsnp_types::{
+	DsnpInnerGraph, DsnpPublicKey, DsnpUserPrivateGraphChunk, DsnpUserPublicGraphChunk,
+};
+use anyhow::Result;
+use apache_avro::{
+	from_avro_datum, from_value, to_avro_datum, to_value, Schema,
+};
+use lazy_static::lazy_static;
+use serde::{Deserialize, Serialize};
+
+lazy_static! {
+	static ref PUBLIC_KEY_SCHEMA: Schema =
+		Schema::parse_str(include_str!("../../../schemas/public_key_schema.json"))
+			.expect("public_key_schema.json is not avro compatible");
+	static ref PUBLIC_GRAPH_CHUNK_SCHEMA: Schema =
+		Schema::parse_str(include_str!("../../../schemas/user_public_graph_chunk.json"))
+			.expect("user_public_graph_chunk.json is not avro compatible");
+	static ref PUBLIC_GRAPH_SCHEMA: Schema =
+		Schema::parse_str(include_str!("../../../schemas/public_graph.json"))
+			.expect("public_graph.json is not avro compatible");
+	static ref PRIVATE_GRAPH_CHUNK_SCHEMA: Schema =
+		Schema::parse_str(include_str!("../../../schemas/user_private_graph_chunk.json"))
+			.expect("user_private_graph_chunk.json is not avro compatible");
+}
+
```

Comment:
you may want to clean it up like this for example

https://github.com/LibertyDSNP/frequency/blob/22aee5f04bfc9ff90a46dc4b9fe5ddcb493c7950/node/service/src/chain_spec/frequency.rs#L112

Reply:
But you are reading and parsing the json anytime `load_frequency_spec()` is called. here we just read and parse it 1 time and can reuse the schema as many times as you want after that.

Reply:
I might be wrong though. let me know what you think

Reply:
Aah thats right, it looks good to me